applications/depth_anything_v2/metadata.json

```diff
@@ -22,10 +22,7 @@
 			"x86_64",
 			"aarch64"
 		],
-		"tags": [
-			"Computer Vision",
-			"Monocular Depth Estimation"
-		],
+		"tags": ["computer vision and perception", "video", "display modes", "TensorRT integration", "holoviz", "depth", "monocular depth estimation", "interactive"],
```

Comment:
"display modes" does not make a lot of sense
"depth" either
"TensorRT integration" either